yr/tutorials/Number_Guessing_Game.md

```diff
@@ -0,0 +1,496 @@
+
+And an example of what you might say to match this utterance is: 
+
+<center> "I guess <span style="background-color: pink">nine</span>."</center>
+
+Let’s add our second **utterance** for the **makeGuessIntent** intent. Another phrase you might respond to Alexa’s question of “What is your guess?” might be something else like “My guess is nine”. Follow the previous steps done to complete our first **utterance**. If done correctly, your **Properties** box should now look like this:
+
+![Add Slot Utterance](../images/number_guessing_game/add_utterances.jpg){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Feel free to create your own **utterances**! (e.g., "**{myGuessSlot}** is my guess”). Remember, the more **utterances** you have, the more likely Alexa will understand correctly.
+
+At this point in our conversation with Alexa we have already invoked the invocation name *guessing game host* and triggered **quizmeIntent**, which causes Alexa to ask us "What is your guess?". We’ve also added the new intent’s **utterances**, “I guess **{myGuessSlot}**" and “My guess is **{myGuessSlot}**.” Remember that **{myGuessSlot}** is a **slot** that represents numbers. To trigger the new *makeGuessIntent* intent, we don’t need to say the invocation name again. Here is a full example of what Alexa might hear when we want to trigger our new intent:
+
+<center> "My guess is <span style ="background-color: pink">ten</span>."</center>
+
+## Define the Intent Handler for *makeGuessIntent* 
+
+Now that we have defined the Interaction Mode of our new **makeGuessIntent**, we need to define the “brain” of the Alexa intent. Our skill can now recognize when we give it our guess, but it doesn’t know what to do or how to handle it. To make the response, we need to define the <span class = "control">when makeGuessIntent spoken </span>block to tell Alexa what to do when you trigger the **“makeGuessIntent”**. To begin, let’s go back to the Blocks view.
+
+![Blocks View](../images/number_guessing_game/image7.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Here, you should see our **makeGuessIntent** as its own Component drawer to the left. Click on that to open it up and drag out the <span class = "control">when makeGuessIntent.spoken </span> block into your workspace.
+
+![makeGuessIntent block](../images/number_guessing_game/image23.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Whenever our Skill hears the user trigger the **makeGuessIntent**, this block will run anything inside of it.
+
+Let's first have our Skill repeat our guess and say: "You guessed **{myGuessSlot}**", where **{myGuessSlot}** represents the value of our guess. Similar to before, we need to use the **say** block from the **Voice** drawer and the <span class = "text">join</span> block from the **Text** drawer. Try implementing everything except **{myGuessSlot}** yourself! Click the hint when you're finished to see what your blocks should look like.
```

Comment:
You say the Hint button but it is a button that reveals the answer, not give a hint.  Answer button?

Reply:
Fixed